app/views/fsss_view.py

```diff
@@ -53,6 +58,23 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
   preferred_name = get_emp_info('preferredName', on_false='None')
 
+  gsuite_filter = {
+      'rangeHigh': 10,
+      'rangeLow': 0,
+      'unique_employee_id': get_emp_info('uniqueEmployeeId'),
+  }
+
+  get_microservice_info = get_microservice_records("gsuitetable",
+                                                   user_filter=gsuite_filter)
+  if str(get_microservice_info.status_code) == '200':
+    gsuite_info = get_microservice_info.json()
+    if len(gsuite_info) > 0:
+      gsuite_info = gsuite_info[0]
+    else:
+      gsuite_info = {}
+  else:
+    gsuite_info = {}
```

Comment:
If we don't get a value back, shouldn't we raise an error?

Reply:
Turns out all this error provisioning is redundant as the test data was unrepresentative of prod - if a person is redirected to this page then there will have  to be a result